packages/paste-website/src/components/tokens-list/token-card/token-example/LineHeightExample.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import {Box} from '@twilio-paste/box';
+import {Text} from '@twilio-paste/text';
+import type {TextProps} from '@twilio-paste/text';
+import {ThemeShape} from '@twilio-paste/theme';
+
+interface TextExampleProps extends Pick<TextProps, 'lineHeight'> {
+  tokenName: string;
+}
+
+export const LineHeightExample: React.FC<TextExampleProps> = ({tokenName, lineHeight}) => {
+  const lineHeightValue = tokenName ? tokenName.toString().match(/\d+/) : null;
+  const fontSizeValue = `fontSize${lineHeightValue}` as keyof ThemeShape['fontSizes'];
+
+  return (
+    <Box backgroundColor="colorBackgroundStrong" flex={1} textAlign="center">
+      <Text as="p" fontSize={fontSizeValue} lineHeight={lineHeight}>
+        Ag
```

Comment:
Suggestion: I would add aria-hidden="true" here because the text is meant to show a color (not to actually provide meaning)